i18n/en.json

```diff
@@ -802,21 +802,13 @@
     "shortName": "Mode",
     "longName": "Operational Mode of Autopilot"
   },
-  "steering.autopilot.targetHeadingNorth": {
-    "shortName": "Target Heading North",
-    "longName": "Target Heading North in Degrees True"
+  "steering.autopilot.targetAngle": {
+    "shortName": "Target (relative) heading",
+    "longName": "TTarget heading for autopilot, relative to magnetic North or Apparent wind +port -starboard"
```

Comment:
Removed the extra T in longname. Since as you say this is not specific to this PR, perhaps we can agree and fix all shortnames in a separate PR?